packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -171,6 +173,24 @@ const DataManagerProvider = ({
     });
   };
 
+  const addCustomFieldAttribute = (
```

Comment:
While I agree the method is a good idea, I'd prefer if you'd change the signature from args to a single object, because it contains so many params. It would make it easier in the context of where it is used to understand what each param means. Wdyt?